Project2015To2017/Analysis/ExtensionMethods.cs

```diff
@@ -1,9 +1,6 @@
-﻿using System.Globalization;
-
-namespace Project2015To2017.Analysis
+﻿namespace Project2015To2017.Analysis
 {
 	internal static class ExtensionMethods
```

Comment:
I decided to keep this one to later move `Transforms.Helpers.GetRelativePathTo` to this class.